src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/Internal/CompilationOptions.cs

```diff
@@ -15,6 +16,7 @@ public class CompilationOptions
         public static readonly string PublicSignTemplate = "--public-sign";
         public static readonly string ApplicationNameTemplate = "--application-name";
         public static readonly string OutputPathTemplate = "--output-path";
+        public static readonly string ViewsToCompileTemplate = "--file";
```

Comment:
I feel like this is an obtuse name, maybe something more like "--content-files" or "--view-files" to be a bit more descriptive and indicate its multi-value nature.